pyveda/models/main.py

```diff
@@ -209,6 +217,8 @@ def _construct_meta(self, name, library, **kwargs):
           "location": kwargs.get("location", {}),
           "training_set": vcp_id,
           "channels_last": self.channels_last
+          # "weights_file_name" = self.weights_file_name,
+          # "model_file_name" = self.model_file_name
```

Comment:
I think we need these here to be live and not commented out, right? The goal to have both of these on the self.meta object. That way they are saved on the model.

Reply:
ahh yes they are back in